src/App.tsx

```diff
@@ -50,14 +50,13 @@ export function App() {
     <ThemeProvider theme={theme}>
       <NavBar />
       <Routes>
-        <Route path="/" element={<HomePage />} />
```

Comment:
Why are we getting rid of this? Wouldn't it make more sense to get rid of "/home" instead? The "home" part seems redundant.

Reply:
so if the path was "/" i wasnt able to style the "Home" in navbar as active link, so i just redirected everything else using NavigateTo to "/home," which would be default so even ofast.io would redirect to ofast.io/home
honestly this was the solution i tried when trying for "/" to be set as active link for "Home" in navbar 
If we get rid of "/home," whole navbar have to change with mapping path to display names, right now i just display path as page name

Reply:
Maybe I'm wrong, but I'm thinking mapping the path shouldn't be much extra code. A dictionary should do it? If possible, we might as well fix that in this diff than delaying it. Lmk what you think.

Reply:
alright i can work on that

Reply:
so instead of dictionary, i just replaced the path for Home page as empty string and when displaying i said if path is empty string I display as "Home" and removed the path "/home"